ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,392 @@
+---
+eip: XXXX
+title: Verification-independent Cross-Chain Messaging Standard
```

Comment:
There are already multiple cross-chain messaging ERCs.  Please consider collaborating.

* ERC-6170: Cross-Chain Messaging Interface
* ERC-7786: Cross-Chain Messaging Gateway
* ERC-7841: Cross-chain Message Format and Mailbox
* RIP-7755: Cross-L2-Call

Reply:
Yup have been in conversations with all these authors about collaboration and potential merging